client/css/base/_tables.css

```diff
@@ -1,20 +1,33 @@
 table {
-    overflow: hidden;
+  overflow: hidden;
+  width: 100%;
+  overflow-x: auto;
+  display: block;
 }
 
-td, th {
-    border: 1px solid #DADFE0;
-    padding: 5px;
+td,
+th {
+  width: 400px;
+  border-bottom: 1px solid $light-grey;
+  padding: 5px;
 }
 
 th {
-    text-align: center;
+  text-align: left;
 }
 
 th.left {
-    text-align: left;
+  text-align: left;
```

Comment:
I'm just saying that it's redundant, as text-align:left is already applied to all THs so it doesn't make sense to keep this class. And when removing classes it's also a good idea to remove usages of those classes in order not to keep unused classes